dbms/src/Interpreters/ActionsVisitor.cpp

```diff
@@ -496,6 +505,18 @@ void ActionsMatcher::visit(const ASTFunction & node, const ASTPtr & ast, Data &
         {
             /// If the argument is not a lambda expression, call it recursively and find out its type.
             visit(child, data);
+
+            // In the above visit() call, if the argument is a literal, we
+            // generated a unique column name for it. Use it instead of a generic
+            // display name.
+            auto child_column_name = child->getColumnName();
+            auto as_literal = dynamic_cast<const ASTLiteral *>(child.get());
+            if (as_literal)
+            {
+                assert(!as_literal->unique_column_name.empty());
+                child_column_name = as_literal->unique_column_name;
```

Comment:
`child_column_name` created, assigned but I cannot find a place where it used.

Reply:
It's in the next couple of lines, maybe the diff didn't load correctly.